source/components-and-apis.rst

```diff
@@ -256,12 +245,6 @@ TCP port 8000), see the following table.
    * - History
      - host:8000/
      - host:8000/history
-   * - EJBCA REST
-     - host:5583/sign
-     - host:8000/sign
-   * - EJBCA REST
-     - host:5583/ca
-     - host:8000/ca
    * - Data Manager
```

Comment:
Please take the opportunity and fix it:

   * - Auth
     - host:5000/revoke
     - host:8000/auth/revoke